docs/architecture/sdk/versioning.md

```diff
@@ -0,0 +1,16 @@
+# Versioning & Breaking Changes
+
+The SDK strives towards maintaining backward compatibility for a reasonable time. This ensures that
```

Comment:
Do we have a specific definition on what we consider reasonable time?

Reply:
No, but minimum until our consuming clients are updated.